salt/states/beacon.py

```diff
@@ -100,8 +100,11 @@ def present(name,
                 ret['comment'].append('Adding {0} to beacons'.format(name))
 
     if save:
-        result = __salt__['beacons.save']()
-        ret['comment'].append('Beacon {0} saved'.format(name))
+        if 'test' in __opts__ and __opts__['test']:
```

Comment:
What about
```suggestion
if __opts__.get('test'):
```
Or if you wanted to be explicit,
```
if __opts__.get('test', False):
```
But (as you're no doubt aware) `.get` defaults to the falsey `None`.